domains/gozar.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "email": "mostafagh1994@gmail.com"
```

add your Github username
```suggestion
    "username": "mos2fagh",
    "email": "mostafagh1994@gmail.com"
```